Localization/Translations/ExtraObjectives/TbcObjectives.lua

```diff
@@ -797,7 +797,7 @@ local tbcObjectiveLocales = {
     ["Collect Cursed Eggs to spawn Malevolent Hatchling"] = { -- 10861
         ["ptBR"] = false,
         ["ruRU"] = false,
-        ["deDE"] = false,
+        ["deDE"] = "Sammelt verfluchte Eier um bösartige Jungtiere zu spawnen.", -- Hat jemand was besseres als spawnen? "Anzulocken" passt hier nicht so wirklich.
```

Comment:
You are translating "spawn" to "anlocken" a few quests below as well. I like that more than "spawn".

Reply:
I thought in the context of gathering eggs it would be irritating to use "anlocken". But ill change that, untill someone has a better idea :)